ui/src/app/app.component.ts

```diff
@@ -1,19 +1,23 @@
-import {Component} from '@angular/core';
+import {Component, OnInit} from '@angular/core';
 import {AuthService} from "./core/auth.service";
 import {Router} from "@angular/router";
+import {CustomIconService} from './core/custom-icon.service';
 
 @Component({
   selector: 'jm-app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
 
   constructor(
     private readonly router: Router,
-    private readonly authService: AuthService
-  ) {}
+    private readonly authService: AuthService,
+    private customIconService: CustomIconService
+  ) { }
 
+  ngOnInit() {
```

Comment:
Deliberate no-op function?

If this function is required because it overrides a default behavior to specify a no-op behavior, it would be great to comment that so it doesn't get deleted by a later maintainer

Reply:
good catch – removed